### PR TITLE
Refine installation cfp intro

### PR DESCRIPTION
**Pull request by SamLR**
try and make it clearer when we people should be talking to us
'officially'

### CODE REVIEW

File: templates/cfp/form_installation_include.html [hunk 1]
```diff
@@ -1,6 +1,14 @@
 <h3>What we're looking for</h3>
 <p>We're looking for installations for our attendees to admire and interact with all over the site. Think big.</p>
 <p>In previous years we've had towers of flame, robots to drag around, trees that react to breathing, visualisations of black holes, fields of light that swarm around you, and domes as far as the eye can see.</p>
+<p>Please fill in this form if any of the following are true of your (proposed) installation:</p>
+<ul>
+  <li>Safety concerns &mdash; if it involves fire, high power lasers, whirling blades etc.</li>
+  <li>Specific siting &mdash; does it need a lot of space, is it very noisy, want it somewhere not in your tent or village or do you want to place it in Null Sector?</li>
```
Comment (marksteward) [inline templates/cfp/form_installation_include.html:7]: This is a bit hard to read. Maybe:

Location &mdash; does it need a lot of space or is it very noisy? Is it best placed outside your village, or do you want it to be part of Null Sector?

**Review by marksteward — APPROVED** [review]

File: templates/cfp/installation_support.html [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+
+<h3>Safety</h3>
+<p>If your installation has significant safety questions, we need to talk.</p>
+<p>If it uses fire, lasers, a lot of power, strange rituals, is very loud, or has people inside it - anything that might present some hazard, please <a href="#contact">contact us</a>. If in doubt, contact us anyway.</p>
+<p>Anything with any kind of safety concern will probably need its own public liability insurance - don't panic, this is easier and cheaper to obtain than you might think. We can help point you at friendly insurers, such as <a href="https://www.axisweb.org/membership/network-associate-with-insurance/">Axis</a>.</p>
+
+<h3>Transport</h3>
+<p>We expect you to bring and set up your thing without our help. If you absolutely need support or specialist equipment, let us know <strong>now</strong> (not a week before) and we’ll see what assistance we can provide. Without that, you should assume that you’ll be doing it all yourself.</p>
+
+<h3>Power</h3>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:42]: `id` attribute please

File: templates/cfp/installation_support.html [hunk 3]
```diff
@@ -0,0 +1,100 @@
...
+<br>
+
+<h2 id="help">When do you <strong>need</strong> to talk to us?</h2> 
+
+<h3>Safety</h3>
+<p>If your installation has significant safety questions, we need to talk.</p>
+<p>If it uses fire, lasers, a lot of power, strange rituals, is very loud, or has people inside it - anything that might present some hazard, please <a href="#contact">contact us</a>. If in doubt, contact us anyway.</p>
+<p>Anything with any kind of safety concern will probably need its own public liability insurance - don't panic, this is easier and cheaper to obtain than you might think. We can help point you at friendly insurers, such as <a href="https://www.axisweb.org/membership/network-associate-with-insurance/">Axis</a>.</p>
+
+<h3>Transport</h3>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:39]: `id` attribute please

File: templates/cfp/installation_support.html [hunk 4]
```diff
@@ -0,0 +1,100 @@
...
+</ul>
+<br>
+
+<h2 id="contact">Contact us</h2>
+<p>Email <a href="mailto:installations@emfcamp.org">installations@emfcamp.org</a> and we'll get back to you as soon as possible</p>
+<br>
+
+<h2 id="help">When do you <strong>need</strong> to talk to us?</h2> 
+
+<h3>Safety</h3>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:34]: `id` attribute please

File: templates/cfp/installation_support.html [hunk 5]
```diff
@@ -0,0 +1,100 @@
...
+<p>If your installation has significant safety questions, we need to talk.</p>
+<p>If it uses fire, lasers, a lot of power, strange rituals, is very loud, or has people inside it - anything that might present some hazard, please <a href="#contact">contact us</a>. If in doubt, contact us anyway.</p>
+<p>Anything with any kind of safety concern will probably need its own public liability insurance - don't panic, this is easier and cheaper to obtain than you might think. We can help point you at friendly insurers, such as <a href="https://www.axisweb.org/membership/network-associate-with-insurance/">Axis</a>.</p>
+
+<h3>Transport</h3>
+<p>We expect you to bring and set up your thing without our help. If you absolutely need support or specialist equipment, let us know <strong>now</strong> (not a week before) and we’ll see what assistance we can provide. Without that, you should assume that you’ll be doing it all yourself.</p>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:40]: can probably leave `absolutely`

File: templates/cfp/installation_support.html [hunk 6]
```diff
@@ -0,0 +1,100 @@
...
+<br>
+
+<h2 id="funding">Funding</h2>
+<p>We have a limited amount of funding available for installations - if you can't afford to make your idea happen, we may be able to help!</p>
+<p>We’re expecting grants to be in the range £100-£500, but larger grants may be possible for exceptional projects.</p>
+<p>If you'd like to apply for funding, please submit your installation to the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call For Participation</a>.</p>
+<br>
+
+
+<h2 id="funding">Tickets</h2>
```
Comment (SamLR) [inline templates/cfp/installation_support.html]: duplicate `id`

File: templates/cfp/installation_support.html [hunk 7]
```diff
@@ -0,0 +1,100 @@
...
+<p>Bringing an installation does not automatically get you a guaranteed EMF ticket, unless we’ve funded it. We have a limited number of guaranteed tickets for exceptional installations available.</p>
+<p>If you'd like to apply for a guaranteed EMF ticket, please submit your installation to the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call For Participation</a>.</p>
+<br>
+
+
+<h2>FAQ</h2>
+
+<h4>I’ve already got a thing, can I just bring it?</h4>
+<p>Yes! You don’t need our agreement, just bring it along. But <a href="#contact">we’d really like to know about it</a>, if only so we can ensure people can find it on-site. And if there is any question at all about safety or logistics, please get in touch.</p>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:74]: could this be something more like `Yes, if it doesn't fall into one of the categories above`

File: templates/cfp/installation_support.html [hunk 8]
```diff
@@ -0,0 +1,100 @@
...
+
+<h4>How do I apply?</h4>
+<p>Fill in the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call for Participation</a>, and you have any questions about the application process, please <a href="#contact">get in touch</a>.</p>
+
+<h4>When can I expect a decision?</h4>
+<p>We’ll be reviewing submissions every few weeks. Funded projects will be awarded on April 4th. If you need money further in advance, please <a href="#contact">contact us</a>.
+
+<h4>What if my thing needs to be indoors?</h4>
+<p>We can host a limited number of installations in the Lounge, the Arcade, and Null Sector. <a href="#contact">Get in touch</a> and we’ll see if we can make it work.</p>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:83]: maybe add `it's also worth getting in contact with one of our villages to see if they can host you`?

File: templates/cfp/installation_support.html [hunk 9]
```diff
@@ -0,0 +1,100 @@
...
+
+<h4>What if it doesn’t work / isn’t lovely / I don’t finish it?</h4>
+<p>We'd like to see enough evidence that the idea is feasible and you can built it before we commit to funding. But if things go wrong, or if you run out of time to build it - don't worry. Just make sure you let us know so we can re-allocate the funding to someone who can use it.</p>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:86]: Is it worth trimming the final sentence to `Just make sure you let us know` as if they've already spent the money we allocated ...

File: templates/cfp/installation_support.html [hunk 10]
```diff
@@ -0,0 +1,100 @@
...
+<br>
+
+
+<h2 id="funding">Tickets</h2>
+<p>Bringing an installation does not automatically get you a guaranteed EMF ticket, unless we’ve funded it. We have a limited number of guaranteed tickets for exceptional installations available.</p>
+<p>If you'd like to apply for a guaranteed EMF ticket, please submit your installation to the <a href="{{ url_for('cfp.form', cfp_type='installation') + ('?closed' if ignore_closed else '') }}">Call For Participation</a>.</p>
+<br>
+
+
+<h2>FAQ</h2>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:71]: `id` attribute?

File: templates/cfp/installation_support.html [hunk 11]
```diff
@@ -0,0 +1,100 @@
...
+
+<h4>I’ve already got a thing, can I just bring it?</h4>
```
Comment (SamLR) [inline templates/cfp/installation_support.html:73]: Can we add a question for `what should I do with my installation at the end of the event` just to make it clear that folks are responsible for taking their things away

**Review by SamLR — COMMENTED** [review]
@Jonty  A load of minor nit picks which I'm not too bothered about (e.g. adding more `id` tags so we can link directly to stuff).

One duplicate `id`.

one specific thing to add: Answer to "what should I do with my installation at the end of the event?" (take it away)

**Pull request merged**